app/scripts/background.js

```diff
@@ -208,6 +205,14 @@ browser.runtime.onConnectExternal.addListener(async (...args) => {
   connectExternal(...args);
 });
 
+function saveTimestamp() {
+  const timestamp = new Date().toISOString();
+
+  browser.storage.session.set({ timestamp }, () => {
```

Comment:
Quick question: could be me overthinking, but what if we clear the session storage, and i think the app will still be running right?